src/Repository/SVNRepository.php

```diff
@@ -67,10 +67,8 @@ public function __construct( SVNRepositoryConfig $repoConfig, IOInterface $io, C
 		$this->vendors = $repoConfig->get( 'vendors' );
 		$this->defaultVendor = key( $this->vendors );
 
-		// set the SvnUtil for all instantiated classes to use
-		if ( !isset( self::$SvnUtil ) ) {
-			self::$SvnUtil = new SvnUtil( $io );
-		}
+		// create an SvnUtil to execute commands
+		$this->SvnUtil = new SvnUtil( $io, $repoConfig->get( 'trust-cert' ) );
```

Comment:
Should it? I think I used titlecase since it's an object...but that might be a borrowed JS convention.